TheWarWithin/DeathKnightFrost.lua

```diff
@@ -1425,9 +1414,9 @@ spec:RegisterAbilities( {
             gain( 1, "runes" )
             gain( 5, "runic_power" )
             applyBuff( "empower_rune_weapon" )
-            for i = 5, 20, 5 do
+            --[[for i = 5, 20, 5 do
```

Comment:
Just delete the commented blocks IMO.

Reply:
I usually leave them while the project isn't finalized, will take out now